### PR TITLE
Fixes #688 Fixed create event button

### LINKED ISSUE #688: Instead of blue "Create" button show blue font
Don't make the "Create" area on the top right too strong. Simply follow the previous design.

screenshot from 2017-08-13 16-11-13

### PR DESCRIPTION
**Pull request by geekychaser**
Fixes #688 Fixed create event button

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
Please Review
Thanks

### CODE REVIEW

File: app/templates/components/nav-bar.hbs [hunk 1]
```diff
@@ -46,7 +46,7 @@
       {{/if}}
       {{#if (not-eq routing.currentRouteName 'create')}}
         <div class="item">
```
Comment (niranjan94) [inline app/templates/components/nav-bar.hbs]: Instead of this, make the whole item a link and add the `ui blue` class here so that it is styled correctly.

**Comment by niranjan94** [comment]
Deployment link: https://pr-786-open-event-frontend-fossasia.surge.sh

File: app/templates/components/nav-bar.hbs (hunk 1 above)
Comment (geekychaser) [inline app/templates/components/nav-bar.hbs]: sir i have not made any changes to package-lock.json but its showing 901 deletion. please help me.
thank you

Comment (niranjan94) [inline app/templates/components/nav-bar.hbs]: Sometimes, when you run `npm install` it might change the file. Just revert the changes to that file in git.

**Comment by niranjan94** [comment]
Deployment link: https://pr-786-open-event-frontend-fossasia.surge.sh

**Comment by geekychaser** [comment]
sir, i have reverted the package-lock.json file.
i have added the ui blue class to div itself hoping that this time it is styled correctly.
please review sir.
thank you.

File: app/templates/components/nav-bar.hbs [hunk 2]
```diff
@@ -45,8 +45,8 @@
         <a href="{{href-to 'login'}}" class="item">{{t 'Login'}}</a>
       {{/if}}
       {{#if (not-eq routing.currentRouteName 'create')}}
-        <div class="item">
-          {{#link-to 'create' class='ui blue button'}}
+        <div class="item ui blue">
```
Comment (niranjan94) [inline app/templates/components/nav-bar.hbs]: Instead of a link-to inside a div, just have a link-to with the class `item ui blue`

Comment (geekychaser) [reply app/templates/components/nav-bar.hbs]: image
sir making whole class link-to is not making 'create event' blue in color due to css property.

Comment (geekychaser) [reply app/templates/components/nav-bar.hbs]: Sir the changes i earlier made using div tag performs and looks sames as this link: https://pr-786-open-event-frontend-fossasia.surge.sh

**Comment by geekychaser** [comment]
sir please check it.

File: app/templates/components/nav-bar.hbs (hunk 2 above)
Comment (niranjan94) [reply app/templates/components/nav-bar.hbs]: `ui blue` class does not exist. In this, it will look blue even without the class. Because blue is the default color of an anchor tag in semantic-ui.

But the issue here is, you're not able to click on any other place other than the text. Because you have put an anchor tag with a `div.item`. 

So, please don't have the `link-to` inside a div instead move the link-to outside of div add a class `ui blue` and style it blue via css.

**Comment by geekychaser** [comment]
[image: Inline image 1]

sir i am not able to pass this test case. Even if my changes are working
perfectly and code is same as expected.

thank you sir

File: app/styles/partials/nav-bar.scss [hunk 3]
```diff
@@ -14,3 +15,7 @@
 .nav-bar-prompt {
   border-radius: 0 !important;
 }
+.blue_text{
+ color:blue !important;
```
Comment (niranjan94) [inline app/styles/partials/nav-bar.scss]: Use the correct semantic ui blue color.

File: app/styles/partials/nav-bar.scss [hunk 4]
```diff
@@ -1,6 +1,7 @@
-.nav-bar > .item > .ui.blue.button {
+.nav-bar > .item > .ui.button {
```
Comment (niranjan94) [inline app/styles/partials/nav-bar.scss]: This should point to that specific item

File: app/styles/partials/nav-bar.scss (hunk 3 above)
Comment (niranjan94) [reply app/styles/partials/nav-bar.scss]: Also keep the class name semantic. Name it `ui blue text`

**Comment by niranjan94** [comment]
Deployment link: https://pr-786-open-event-frontend-fossasia.surge.sh

**Comment by geekychaser** [comment]
sir , is it now working properly or some changes are being required?


On Fri, Sep 1, 2017 at 1:41 AM, Niranjan Rajendran <notifications@github.com

**Comment by niranjan94** [comment]
Deployment link: https://pr-786-open-event-frontend-fossasia.surge.sh

File: app/styles/partials/nav-bar.scss [hunk 5]
```diff
@@ -14,3 +14,7 @@
 .nav-bar-prompt {
   border-radius: 0 !important;
 }
+
+.blue.text {
+  color: #425ff4 !important;
```
Comment (geekyd) [inline app/styles/partials/nav-bar.scss]: @geekychaser Please use `#2185D0` here

**Comment by niranjan94** [comment]
Deployment link: https://pr-786-open-event-frontend-fossasia.surge.sh

File: app/templates/components/nav-bar.hbs [hunk 6]
```diff
@@ -19,10 +19,8 @@
       {{/if}}
       {{#if session.isAuthenticated}}
         {{#ui-dropdown class='item' action='hide'}}
-          <img class="ui avatar round image nav-bar-image"
-               alt="{{authManager.currentUser.email}}"
-               src="{{if authManager.currentUser.avatarUrl authManager.currentUser.avatarUrl
-                         '/images/placeholders/avatar.png'}}">
+          {{widgets/safe-image class='ui avatar round image nav-bar-image' src=(if authManager.currentUser.avatarUrl authManager.currentUser.avatarUrl
```
Comment (geekyd) [inline app/templates/components/nav-bar.hbs]: @geekychaser This change has already been made, seems like your branch is out of date. Can you manually rebase the branch?

Comment (geekyd) [reply app/templates/components/nav-bar.hbs]: The changes related to this were made in https://github.com/fossasia/open-event-frontend/pull/814

**Comment by niranjan94** [comment]
Deployment link: https://pr-786-open-event-frontend-fossasia.surge.sh

File: app/templates/components/nav-bar.hbs (hunk 6 above)
Comment (geekychaser) [reply app/templates/components/nav-bar.hbs]: sir rebased the branch and commited. Please review it.

thank you

**Comment by geekychaser** [comment]
sir ,
please review it and suggest me if some more changes are required.

**Comment by geekyd** [comment]
@geekychaser Just squash your commits. It looks good.

**Comment by niranjan94** [comment]
Deployment link: https://pr-786-open-event-frontend-fossasia.surge.sh

**Comment by geekychaser** [comment]
sir,
all the commits are being squashed. please have a look.
thank you

**Comment by niranjan94** [comment]
Deployment link: https://pr-786-open-event-frontend-fossasia.surge.sh
